### PR TITLE
fix(list): fix overlap in codebase list

### PR DESCRIPTION
**Pull request by mindreeper2420**
fix the overlap in the codebase list due to adding a new element once a che workspace is created

### CODE REVIEW

**Comment by mindreeper2420** [comment]
**Before Fix:**
screen shot 2017-06-22 at 11 50 56 am

**After Fix:**
screen shot 2017-06-22 at 11 50 45 am

File: src/app/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.less [hunk 1]
```diff
@@ -14,4 +14,5 @@
   height: 32px;
   margin-right: 5px;
   width: 200px;
+  display: inline-flex;
```
Comment (joshuawilson) [inline src/app/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.less]: Why are you adding flex back in?

Comment (mindreeper2420) [inline src/app/create/codebases/codebases-item-workspaces/codebases-item-workspaces.component.less]: @joshuawilson my bad - should be inline-block. I'll push the fix.

**Comment by fabric8cd** [comment]
@mindreeper2420 snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1579-fabric8-ui.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@mindreeper2420 snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1579-fabric8-ui.int.rdu2c.fabric8.io

**Pull request merged**